prac_06/guitars.py

```diff
@@ -25,12 +26,15 @@ def main():
     maximum_cost_length = max(len(str(f"{guitar.cost:,.2f}")) for guitar in guitars)
     # print(maximum_name_length)
     # print(maximum_cost_length)
-    guitars.sort(key=attrgetter("year"))
-    for i, guitar in enumerate(guitars, 1):
-        vintage_display = " (vintage)" if guitar.is_vintage() else ""
-        print(
-            f"Guitar {i}: {guitar.name:>{maximum_name_length}} ({guitar.year:{MAXIMUM_YEAR_LENGTH}}), "
-            f"worth $ {guitar.cost:>{maximum_cost_length},.2f}{vintage_display}")
+    if guitars:
+        guitars.sort(key=attrgetter("year"))
+        for i, guitar in enumerate(guitars, 1):
+            vintage_display = " (vintage)" if guitar.is_vintage() else ""
```

Comment:
I don't believe you need an else statement as it is a boolean value.

Reply:
I think else is required for ternary operator. If there is no else and guitar is not vintage, vintage_display variable is undefined (not an empty string). 
If I have initiated vintage_display = "" before if statement, then else is not required like below. What do you think?
vintage_display = ""
if guitar.is_vintage():
    vintage_display = " (vintage)"